scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ServerHandshaker.java

```diff
@@ -676,26 +683,32 @@ private void createCertificateRequest(final ClientHello clientHello, final DTLSF
 	private byte[] receivedClientKeyExchange(final PSKClientKeyExchange message) throws HandshakeException {
 
 		clientKeyExchange = message;
-		byte[] psk = null;
 
-		// use the client's PSK identity to get right preshared key
+		// use the client's PSK identity to look up the pre-shared key
 		String identity = message.getIdentity();
-
-		LOGGER.debug("Client [{}] uses PSK identity [{}]",
-				new Object[]{getPeerAddress(), identity});
+		byte[] psk = pskStore.getKey(getIndicatedServerNames(), identity);
+		String virtualHost = null;
 
 		if (getIndicatedServerNames() == null) {
-			psk = pskStore.getKey(identity);
+			LOGGER.debug("Client [{}] uses PSK identity [{}]",
+					new Object[]{getPeerAddress(), identity});
```

Comment:
Recently, we don't use the explicit `new Object[]{ ... }`

Reply:
are we following any *good practice* as to which places we correct it and which not? When you guys work on a class, do you replace all occurrences with the log4j based syntax or just in those places that you are working on anyway?

Reply:
AFAIK we don't apply a "good practice". 
But for me it seems to be a new logging statements so I would go for the style without `new Object[]{ ... }`.

Reply:
ok, then I will replace it in the code that I have *touched* and leave the rest *as-is*